mods/cnc/rules/structures.yaml

```diff
@@ -821,12 +822,12 @@ ATWR:
 		Prerequisites: anyhq, ~techlevel.medium
 		Queue: Defence.GDI
 	Building:
-		Footprint: _ x
-		Dimensions: 1,2
+		Footprint: x
+		Dimensions: 1,1
```

Comment:
When I tested this the missiles came from tabove the roof instead of from the windows or the launchers below the windows.